sources/install/package_ad.sh

```diff
@@ -660,10 +660,7 @@ function install_pywsus() {
 
 function install_donpapi() {
     colorecho "Installing DonPAPI"
-    git -C /opt/tools/ clone --depth 1 https://github.com/login-securite/DonPAPI.git
-    cd /opt/tools/DonPAPI
-    python3 -m venv ./venv/
-    ./venv/bin/python3 -m pip install -r ./requirements.txt
+    python3 -m pipx install git+https://github.com/login-securite/DonPAPI
     add-aliases donpapi
```

Comment:
Can you delete the alias too ? https://github.com/ThePorgs/Exegol-images/blob/dev/sources/assets/zsh/aliases.d/donpapi